packages/editor/src/store/actions.js

```diff
@@ -724,17 +724,34 @@ export function removeEditorPanel( panelName ) {
  *                                              opened (true) or closed (false).
  *                                              To specify an insertion point,
  *                                              use an object.
- * @param {string}         value.rootClientId   The root client ID to insert at.
- * @param {number}         value.insertionIndex The index to insert at.
+ * @param {string}         value.rootClientId   Deprecated. The root client ID to insert at.
+ * @param {number}         value.insertionIndex Deprecated. The index to insert at.
+ * @param {string}         value.filterValue    A query to filter the inserter results.
+ * @param {Function}       value.onSelect       A callback when an item is selected.
+ * @param {string}         value.tab            The tab to open in the inserter.
+ * @param {string}         value.category       The category to initialize in the inserter.
  *
  * @return {Object} Action object.
  */
-export function setIsInserterOpened( value ) {
-	return {
-		type: 'SET_IS_INSERTER_OPENED',
-		value,
+export const setIsInserterOpened =
+	( value ) =>
+	( { dispatch, registry } ) => {
+		if (
+			typeof value === 'object' &&
+			value.hasOwnProperty( 'rootClientId' ) &&
+			value.hasOwnProperty( 'insertionIndex' )
+		) {
+			return registry.dispatch( blockEditorStore ).setInsertionPoint( {
+				rootClientId: value.rootClientId,
+				index: value.insertionIndex,
+			} );
```

Comment:
There are two bugs here:
1. It shouldn't return here, it should just dispatch it.
2. `setInsertionPoint` isn't an available action because it's in the private-actions in the block editor store. Are we allowed to use it in this editor package if it's private in the block editor package?